bitnami/grafana/values.yaml

```diff
@@ -568,6 +568,26 @@ imageRenderer:
   ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
   ##
   affinity: {}
+  ## Grafana Image Renderer containers' resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param resources.limits The resources limits for Grafana containers
```

Comment:
Could you modify resources.limits to imageRenderer.resources.limits, please? To fix the "Generate Chart Readme / generate-chart-readme" repository test.

Reply:
Fixed